src/index.ts

```diff
@@ -45,8 +45,10 @@ interface BackendParams {
 	app: express.Express | string;
 }
 
+type Super = PinejsClientCore<unknown>;
+
 /** A pine testing client fused with the api of supertest */
-export class PineTest extends PinejsClientCore<PineTest> {
+export class PineTest extends PinejsClientCore<unknown> {
```

Comment:
This is no longer required and is unused in the latest pinejs-client, so changing it to `unknown` just matches what pinejs-client would now default to (I realized technically I can probably remove the generic altogether but then I'd just have to add it back for model based typing support)